pegasus/routes/v2_forms_routes.rb

```diff
@@ -61,7 +61,7 @@
 
 review '/v2/forms/:kind/:secret' do |kind, secret|
   dont_cache
-  forbidden! unless dashboard_user && dashboard_user[:admin]
+  forbidden! unless dashboard_user
```

Comment:
I did a search to be sure, this also uses review: https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/views/class_review.haml#L66

Not sure of the context of how it's used though